official/benchmark/models/resnet_imagenet_main.py

```diff
@@ -25,6 +25,7 @@
 from absl import logging
 import tensorflow as tf
 
+from tensorflow_model_optimization.python.core.clustering.keras import cluster
 import tensorflow_model_optimization as tfmot
```

Comment:
We didn't do this previously, but should start doing it now and the long term integration:

It'd be best to import `tfmot` only when the flags indicate that tfmot will be used, so that if there were hypothetically breakages, the rest of the benchmarks could continue to run.
   

So only if flags_obj.clustering_method

Reply:
Unresolved still.

If my comment wasn't clear, the idea is that the entire "import tensorflow_model_optimization" should be moved under an if statement and done only if clustering or pruning are enabled